doc/spec_exceptionHandling.dox

```diff
@@ -69,12 +71,18 @@ namespace ChimeraTK {
 
 \subsection spec_execptionHandling_behaviour_recovery Recovery
 - 3. The framework tries to resolve an exception state by periodically re-opening the faulty device.
-  - \anchor b_3_1 3.1 After successfully re-opening the device, a recovery procedure is executed before allowing any read/write operations from the ApplicationModules and FanOuts again. This recovery procedure involves:
-    - \anchor exceptionHandling_b_3_1_1 3.1.1 the execution of so-called initialisation handlers (see \ref exceptionHandling_b_3_2 "3.2") [\ref testExceptionHandling_b_3_1_1 "T"], and
-    - \anchor exceptionHandling_b_3_1_2 3.1.2 restoring all registers that have been written since the start of the application with their latest values. The register values are restored in the same order they were written. Registers of the type ChimeraTK::Void are not written. \ref comment_b_3_1_2 "(*)" [\ref testExceptionHandling_b_3_1_2 "T"]
+  - \anchor b_3_0 3.0 In a recovery group (see \ref exceptionHandling_a_5_1 "A.5.1"), devices wait until all involved devices have seen the error condition before trying to re-open ("barrier DETECT"). [not testable?]
```

Comment:
"devices" -> "DeviceManager" (also below a number of times, I will not repeat this comment everywhere...)

Reply:
The barrier names are a bit ambiguous since it is not clear if "DETECT" is before or after the detection. I suggest either "DETECTED" and "OPENED" etc, or "POST-DETECT" and "POST-OPEN".